napari/layers/points/points.py

```diff
@@ -2054,12 +2054,13 @@ def _move(
                 self.data[np.ix_(selection_indices, disp)] + shift
             )
             self.refresh()
-        self.events.data(
-            value=self.data,
-            action=ActionType.CHANGED,
-            data_indices=tuple(selection_indices),
-            vertex_indices=((),),
-        )
+            if not self.events.blocked():
```

Comment:
Should this be `self.events.data.blocked()`

Reply:
Why this if? 
Blocker is to not need to check these thing.

Reply:
ah I thought this deletion was already pushed. This was when I was checking whether I could block the second emission of this event in the test.

Reply:
Partially resolved [59967e9](https://github.com/napari/napari/pull/6611/commits/59967e9b4eac736a96aab5ede3a1090ed02388e3). However, in the test the `CHANGED` event is not blocked during the `mouse_move` event causing it to be emitted once while applying this callback and once after the `mouse_release`. This is also the case when using `MagicMock`.